tests/test_filesystem.py

```diff
@@ -96,3 +99,55 @@ def test_filesystem_zip__with_source_prefix():
     for source_file in source_files_in_zip:
         with fs.open(source_file) as f:
             assert hasattr(f, 'read')
+
+
+def test_filesystem_git():
+    import pycobertura.filesystem as fsm
+    from pycobertura.filesystem import GitFileSystem
+
+    branch, folder, filename = "master", "tests/dummy", "test-file"
+
+    with patch.object(fsm, "subprocess") as subprocess_mock:
+        subprocess_mock.check_output = MagicMock(return_value=b"<file-content>")
+
+        fs = GitFileSystem(folder, branch)
```

Comment:
Non-blocking: could we reuse the import `fsm` and avoid the import `from pycobertura.filesystem import GitFileSystem` above?

```suggestion
        fs = fsm.GitFileSystem(folder, branch)
```